ide/projectui/src/org/netbeans/modules/project/ui/OpenProjectList.java

```diff
@@ -441,12 +441,18 @@ boolean closeBeforeOpen(final Project[] arr) {
                 }
             });
         }
-            
+
+        @NbBundle.Messages({
+            "LOAD_PROJECTS_ON_START=true"
+        })
         private void loadOnBackground() {
-            lazilyOpenedProjects = new ArrayList<Project>();
-            List<URL> URLs = OpenProjectListSettings.getInstance().getOpenProjectsURLs();
-            final List<Project> initial = new ArrayList<Project>();
-            final LinkedList<Project> projects = URLs2Projects(URLs);
+            lazilyOpenedProjects = new ArrayList<>();
+            final boolean loadProjectsOnStart = "true".equals(Bundle.LOAD_PROJECTS_ON_START());
```

Comment:
It's an interesting usage of resource bundle. :-)
But I think it's rather a hack to use module resource bundle for configuration.
Do we already use such a pattern somewhere?
If not I would prefer a method.

Reply:
Yes, it is used at many places, controlling behavior of splash screen, controlling behavior of window system. We even have `<api group='branding'..../>` for that case - here is the current [list of branding APIs](https://bits.netbeans.org/dev/javadoc/branding.html).